Tools/LyTestTools/ly_test_tools/mobile/android.py

```diff
@@ -149,7 +149,7 @@ def pull_files_to_pc(package_name, logs_path, device=None):
         if 'does not exist' in err.output:
             logger.info('Could not pull logs since none exist on device {}'.format(device))
         else:
-            raise
+            raise exceptions.LyTestToolsFrameworkException(err)
```

Comment:
Instead of constructing one exception from another, I recommend reraising with:
`raise exceptions.LyTestToolsFrameworkException() from err`

This should automatically include the underlying exception's stacktrace with proper reporting.